src/pages/what_love_picnic.adoc

```diff
@@ -37,7 +37,7 @@ belongs to you as much as to anyone else.
 
 Code reviews are at the core of our development, but it's not always
 fun. We want everything to be interesting so we've put code reviews on
-the blockchain. You can find out more in this article.
+the blockchain. You can find out more in https://blog.picnic.nl/crypto-incentives-for-code-reviews-71a0be53d130[this article].
```

Comment:
We have started to externalize our links. IIUC we can add this one to `links.adoc`. A link ID could be `:url-picnic-blog-incentives-for-code-reviews:`

Reply:
@Stephan202 resolved 👍